save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestSuitesService.kt

```diff
@@ -37,22 +37,31 @@ class TestSuitesService {
     @Suppress("UnsafeCallOnNullableType")
     fun saveTestSuite(testSuitesDto: List<TestSuiteDto>): List<TestSuite> {
         val testSuites = testSuitesDto
-            .map { TestSuite(it.type, it.name, it.project, null, it.propertiesRelativePath, it.testSuiteRepoUrl) }
+            .distinctBy {
```

Comment:
don't understand the case for distinct

I understand when we could have the duplicate by name and description, but what's with the path of suite

Reply:
There may be suites with same names in the same repo, but with different test roots. Then we should save them separately. But if everything is same, other than description, it's probably the same test suite (do we have any checks in SAVE that description shouldn't be overriden if test suite name is the same? I think it would make sense).

Reply:
so-so, but ok